torch/onnx/symbolic.py

```diff
@@ -554,16 +555,12 @@ def Elman_RNN_symbolic_builder(
     if bidirectional:
         return _unimplemented("RNN", "bidirectional")
 
-    def symbolic(g, input, all_weights, h0, **fkwargs):
-        # TODO elide this argument to increase parametricity. This is
-        # nontrivial because we provide subsequent optional arguments,
-        # and ONNX does not have a mechanism for skipping non-trailing
-        # optional arguments.
-        sequence_len, batch_size = input.type().sizes()[0:2]
-        sequence_lens = g.op('Constant', value_t=torch.IntTensor(batch_size).fill_(sequence_len))
-
+    def symbolic(g, input, all_weights, h0, batch_sizes):
         prev_output = input
         h_outs = []
+
+        sequence_lens = g.op("Undefined") if batch_sizes is None else batch_sizes
```

Comment:
nit: maybe declare `g.op(Undefined)` as 'unused' somewhere to make it look nicer?